pkg/keyspace/keyspace.go

```diff
@@ -46,6 +46,10 @@ const (
 	regionLabelIDPrefix = "keyspaces/"
 	// regionLabelKey is the key for keyspace id in keyspace region label.
 	regionLabelKey = "id"
+	// UserKindKey is the key for user kind in keyspace config.
+	UserKindKey = "user_kind"
+	// TSOKeyspaceGroupIDKey is the key for tso keyspace group id in keyspace config.
+	TSOKeyspaceGroupIDKey = "tso_keyspace_group_id"
```

Comment:
Suggest reflecting the hierarchy from the naming, so maybe `KeyspaceGroupIDTSOKey` is a better name.

Reply:
But it sounds strange. lol